launchflow/gcp/static_site.py

```diff
@@ -0,0 +1,103 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from launchflow import exceptions
+from launchflow.gcp.gcs import BackendBucket
+from launchflow.gcp.service import GCPStaticService
+from launchflow.models.enums import ServiceProduct
+from launchflow.node import Inputs
+from launchflow.resource import Resource
+from launchflow.service import DNSOutputs, DNSRecord, StaticServiceOutputs
+
+
+@dataclass
+class StaticSiteInputs(Inputs):
+    pass
+
+
+class StaticSite(GCPStaticService):
```

Comment:
nit: add link to external docs if available. Also I would add what this actually spin up since it's much more than just a bucket (e.g. load balancer)

Reply:
saving for follow up